idea/idea-fir/src/org/jetbrains/kotlin/idea/quickfix/fixes/ReplaceCallFixFactories.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.quickfix.fixes
+
+import com.intellij.psi.PsiElement
+import org.jetbrains.kotlin.idea.KotlinBundle
+import org.jetbrains.kotlin.idea.fir.api.applicator.HLApplicatorInput
+import org.jetbrains.kotlin.idea.fir.api.applicator.applicatorByQuickFix
+import org.jetbrains.kotlin.idea.fir.api.fixes.HLApplicatorWithTargetAndInput
+import org.jetbrains.kotlin.idea.fir.api.fixes.diagnosticFixFactory
+import org.jetbrains.kotlin.idea.fir.api.fixes.withInput
+import org.jetbrains.kotlin.idea.frontend.api.fir.diagnostics.KtFirDiagnostic
+import org.jetbrains.kotlin.idea.frontend.api.types.KtTypeNullability
+import org.jetbrains.kotlin.idea.frontend.api.types.KtTypeWithNullability
+import org.jetbrains.kotlin.idea.quickfix.ReplaceCallFix
+import org.jetbrains.kotlin.idea.quickfix.ReplaceImplicitReceiverCallFix
+import org.jetbrains.kotlin.idea.quickfix.ReplaceWithSafeCallFix
+import org.jetbrains.kotlin.psi.*
+
+object ReplaceCallFixFactories {
+    private val replaceWithSafeCallFixApplicator =
+        applicatorByQuickFix<KtExpression, Input, ReplaceCallFix>(getFamilyName = { KotlinBundle.message("replace.with.safe.call") },
```

Comment:
There is a `KotlinBundle.lazyMessage` for that
So, `getFamilyName = { KotlinBundle.message("replace.with.safe.call") }` -> `getFamilyName = KotlinBundle.lazyMessage("replace.with.safe.call")`

Reply:
Also, please consider placing such arguments on new lines:
```
applicatorByQuickFix<KtExpression, Input, ReplaceCallFix>(
   getFamilyName = KotlinBundle.lazyMessage("replace.with.safe.call"),
   isApplicableByPsi = { psi -> psi is KtDotQualifiedExpression }
) { psi, input ->
   ReplaceWithSafeCallFix(psi as KtDotQualifiedExpression, input.notNullNeeded)
}
```